api/src/main/java/org/openmrs/User.java

```diff
@@ -73,9 +76,9 @@
 @AttributeOverrides({
         @AttributeOverride(name = "name", column = @Column(table = "users_unused_fields", name = "name", insertable = false, updatable = false)),
         @AttributeOverride(name = "description", column = @Column(table = "users_unused_fields", name = "description", insertable = false, updatable = false)) })
-public class User extends BaseChangeableOpenmrsMetadata implements java.io.Serializable, Attributable<User> {
+public class User extends BaseOpenmrsObject implements Auditable, Retireable {
```

Comment:
So, rather than remove `Serializable` and `Attributable<User>` from here, we want to add `Auditable` and `Retireable` so that the whole signature looks like:

```java
public class User extends BaseOpenmrsObject implements Serializable, Attributable<User>, Auditable, Retireable {
...
}
```